niceday_client/tests/test_niceday_client.py

```diff
@@ -36,3 +36,13 @@ def test_get_profile(mock_niceday_api):
     for k in USER_PROFILE_KEYS:
         assert k in profile
         assert isinstance(profile[k], str)
+
+
+@pytest.mark.integration
+def test_post_message():
+    """
+    Test posting a message to the Niceday server
+    """
+    client = NicedayClient()
+    existing_user_id = 38527
+    client.post_message(existing_user_id, 'Hello world')
```

Comment:
No need to add a unit test, as it would not test anything useful. But this is useful for testing during development.

Reply:
That makes sense to me - it's marked `@pytest.mark.integration` anyway so won't cause problems